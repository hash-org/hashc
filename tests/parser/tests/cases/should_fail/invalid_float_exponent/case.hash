let k = 12e23.0;
