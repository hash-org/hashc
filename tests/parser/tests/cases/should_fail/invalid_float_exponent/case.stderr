error: Failed to parse
 --> $DIR/case.hash:1:12
1 |   k := 12e23.0;
  |              ^ here
  = note: Expected field name access or an infix function call.
